rs/types/management_canister_types/src/lib.rs

```diff
@@ -1006,6 +1029,14 @@ impl CanisterStatusResultV2 {
         controller: PrincipalId,
         controllers: Vec<PrincipalId>,
         memory_size: NumBytes,
+        wasm_memory_size: NumBytes,
+        stable_memory_size: NumBytes,
+        global_memory_size: NumBytes,
+        wasm_binary_memory_size: NumBytes,
+        custom_sections_memory_size: NumBytes,
+        canister_history_memory_size: NumBytes,
+        wasm_chunk_store_memory_size: NumBytes,
+        snapshot_memory_size: NumBytes,
```

Comment:
Nit: the argument names here might need to be adjusted after the renaming (just for consistency reasons). Same goes for some of the tests defined in the other file, they use the "old" names so we might want to update them for consistency.

Reply:
Most of them should be updated now! Let me know if I missed anything.